dataplicity/m2m/bencode.py

```diff
@@ -60,12 +61,33 @@ def encode(obj):
 
     def add_encode(obj):
         if isinstance(obj, bytes):
-            append(b"{}:{}".format(len(obj), obj))
+            if PY2:
+                append(b"{}:{}".format(len(obj), obj))
+            elif PY3:
+                append(six.b("{}:".format(len(obj))))
```

Comment:
I think that the only reason why I did this was that `b"{}".format()` would not work in python3, and I wanted to keep the code as close as possible to the original. However, the code suggested by @willmcgugan solves this problem, so I'm replacing it now.